src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,21 +2,25 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LoggerFactory.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login was called. Params: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed for user: {}", login);
             throw new AuthenticationException("Username or password are incorrect");
         }
         return user;
     }
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
         return user;
```

Comment:
The `findByLogin` method appears to return a hardcoded user, which is not suitable for a production environment. Ensure that the method actually queries a data source to retrieve the user based on the provided login.